CHANGELOG.md

```diff
@@ -9,7 +9,8 @@
 * Fix a data race when opening a flexible sync Realm (since v12.1.0).
 * Fixed a missing backlink removal when setting a Mixed from a TypedLink to null or any other non-link value. Users may have seen exception of "key not found" or assertion failures such as `mixed.hpp:165: [realm-core-12.1.0] Assertion failed: m_type` when removing the destination link object. ([#5574](https://github.com/realm/realm-core/pull/5573), since the introduction of Mixed in v11.0.0)
 * Asymmetric sync now works with embedded objects. (Issue [#5565](https://github.com/realm/realm-core/issues/5565), since 12.1.0)
- 
+* In Realm JS, the client reset callback can result in the fatal error `Realm accessed on incorrect thread`. Using a thread safe reference instead of Realm instance can midigate the issue. ([realm/realm-js#4410](https://github.com/realm/realm-js/issues/4410), since v11.7.0)
```

Comment:
*mitigate
You could argue this is a fix, but it might warrant being under the breaking changes section since SDKs will have to make changes to adopt this.

Reply:
I have moved it